# This file defines the CSM
vehicle_description:
  Ixx: 40823.10274676924          # moment  of inertia about i^b in Kg*m^2
  Iyy: 90593.63634466067          # moment  of inertia about j^b in Kg*m^2
  Izz: 98743.00497284240          # moment  of inertia about k^b in Kg*m^2
  Ixy: 1537.81116196125           # product of inertia about ij  in Kg*m^2
  Ixz: -3179.30485957706          # product of inertia about ik  in Kg*m^2
  Iyz: 128.57742049302            # product of inertia about jk  in Kg*m^2
